tests/algolia/rules_engine_test.py

```diff
@@ -283,7 +292,52 @@ def test_should_return_false_when_stocks_beginning_datetime_have_not_changed(sel
         result = is_eligible_for_reindexing(offer=offer,
                                             offer_details={'name': 'super offre',
                                                            'dateRange': ['2020-01-01 00:00:00', '2020-01-02 00:00:00'],
-                                                           'dates': [1577836800.0]})
+                                                           'dates': [1577836800.0],
+                                                           'prices': [10.0]})
+
+        # Then
+        assert result is False
+
+    @clean_database
+    def test_should_return_true_when_stocks_prices_have_changed(self, app):
+        # Given
+        offerer = create_offerer(is_active=True, validation_token=None)
+        venue = create_venue(offerer=offerer, validation_token=None)
+        offer = create_offer_with_event_product(event_name='super offre', venue=venue)
+        stock = create_stock(beginning_datetime=datetime(2020, 1, 1),
+                             end_datetime=datetime(2020, 1, 2),
+                             offer=offer,
+                             price=10)
+        repository.save(stock)
+
+        # When
+        result = is_eligible_for_reindexing(offer=offer,
+                                            offer_details={'name': 'super offre',
+                                                           'dateRange': ['2020-01-01 00:00:00', '2020-01-02 00:00:00'],
+                                                           'dates': [1577836800],
+                                                           'prices': [11.0]})
 
         # Then
-        assert result is False
\ No newline at end of file
+        assert result is True
```

Comment:
```suggestion
        assert result
```
Et j'en profiterais pour renommer result en `is_offer_eligible` non ?

Reply:
👍